backend/seeds/seeds.go

```diff
@@ -244,6 +244,10 @@ func setupClusterRoles(store store.Store) error {
 				Verbs:     []string{"get", "update"},
 				Resources: []string{types.LocalSelfUserResource},
 			},
+			types.Rule{
+				Verbs:     []string{"list"},
```

Comment:
Should this apply to GET as well?

Reply:
Good question; I think `list` is enough to solve our problem. `get` would let you retrieve specific namespaces objects, which are [pretty uninteresting](https://github.com/sensu/sensu-go/blob/master/api/core/v2/namespace.proto#L15-L19).

Reply:
I mentioned this in Slack, but I think we should add get as well.